src/Controls/src/Core/Shell/Shell.cs

```diff
@@ -94,77 +121,207 @@ static void OnFlyoutItemIsVisibleChanged(BindableObject bindable, object oldValu
+        /// </summary>
+        public static readonly BindableProperty ItemTemplateProperty =
 			BindableProperty.CreateAttached(nameof(ItemTemplate), typeof(DataTemplate), typeof(Shell), null, BindingMode.OneTime);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetItemTemplate']/Docs/*" />
-		public static DataTemplate GetItemTemplate(BindableObject obj) => (DataTemplate)obj.GetValue(ItemTemplateProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetItemTemplate']/Docs/*" />
-		public static void SetItemTemplate(BindableObject obj, DataTemplate itemTemplate) => obj.SetValue(ItemTemplateProperty, itemTemplate);
+        /// <summary>
+        /// Gets the <see cref = "DataTemplate" /> applied to each <see cref = "FlyoutItem" /> object managed by Shell.
+        /// </summary>
+        /// <param name="obj">The object that sets the DataTemplate applied to Item objects.</param>
+        /// <returns>The DataTemplate applied to Item objects.</returns>
+        public static DataTemplate GetItemTemplate(BindableObject obj) => (DataTemplate)obj.GetValue(ItemTemplateProperty);
+
+        /// <summary>
+        /// Sets the <see cref = "DataTemplate" /> applied to each <see cref = "FlyoutItem" /> object managed by Shell.
+        /// </summary>
+        /// <param name="obj">The object that sets the DataTemplate applied to Item objects.</param>
+        /// <param name="itemTemplate">The DataTemplate applied to Item objects.</param>
+        public static void SetItemTemplate(BindableObject obj, DataTemplate itemTemplate) => obj.SetValue(ItemTemplateProperty, itemTemplate);
 
 		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetBackButtonBehavior']/Docs/*" />
 		public static BackButtonBehavior GetBackButtonBehavior(BindableObject obj) => (BackButtonBehavior)obj.GetValue(BackButtonBehaviorProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetBackButtonBehavior']/Docs/*" />
+
+		/// <summary>
+		/// Sets the behavior of the back button.
+		/// </summary>
+		/// <param name="obj">The object that sets the behavior of the back button.</param>
+		/// <param name="behavior">The back button behavior.</param>
 		public static void SetBackButtonBehavior(BindableObject obj, BackButtonBehavior behavior) => obj.SetValue(BackButtonBehaviorProperty, behavior);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetPresentationMode']/Docs/*" />
+		/// <summary>
+		/// Gets the navigation animation that occurs when a page is navigated to with the <see cref = "GoToAsync(ShellNavigationState, bool)" /> method.
+		/// </summary>
+		/// <param name="obj">The object that modifies the Shell tabs visibility.</param>
```

Comment:
I think you can just avoid saying "Shell tabs visibility" and just say "tabs visibility", since its implied you're talking about shell in the documentation.

Reply:
Changed.